.github/workflows/pypipublish.yaml

```diff
@@ -65,6 +65,27 @@ jobs:
           python -m pip install dist/roms_tools*.whl
           python -c "import roms_tools; print(roms_tools.__version__)"
 
+  upload-to-test-pypi:
+    needs: test-built-dist
+    if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
```

Comment:
Does it make sense to loosen this constraint?

Reply:
It would be great to uncover a potential uploading to PyPI issue before merging the PR. Is that possible?

Reply:
I removed the constraint, altogether. I misunderstood the amount of manual commits to main, so the existing constraints on the whole step should do the trick. I am curious how `setuptools_scm` generates version numbers without a release. Might need a tweak, still.

Reply:
Moving test pypi release to a new ticket to unblock this